docs/sources/operators-guide/monitor-grafana-mimir/collecting-metrics-and-logs.md

```diff
@@ -107,6 +114,26 @@ Older versions of the Helm chart need to be manually instrumented. This means th
 that collects logs and metrics from Mimir or GEM. To set up Grafana Agent,
 see [Set up Grafana Agent](https://grafana.com/docs/agent/latest/set-up/).
```

Comment:
```suggestion
see [Set up Grafana Agent](https://grafana.com/docs/agent/latest/set-up/). Once your Agent is deployed, use the [example Agent configuration](#example-agent-configuration) in the section below to configure the Agent to scrape Mimir (or GEM). 
```

Reply:
Added a link to the actual section.

```suggestion
see [Set up Grafana Agent](https://grafana.com/docs/agent/latest/set-up/). Once your Agent is deployed, use the [example Agent configuration](#example-agent-configuration) in the section [Collect metrics and logs via the Helm chart](#collect-metrics-and-logs-via-the-helm-chart) to configure the Agent to scrape Mimir or GEM.
```

Reply:
I misread where your suggesteion was posted. I corrected the mistake in 70bc4c8b7835ad388fd92db7e6066cf73cc59b60

some advice I've gotten from tech writers is to not use "above" and "below" because the sections may not be ordered vertically for some readers. Since the link already points to the section, I just removed the word "section"